packages/core/src/utils/storage/s3-storage.ts

```diff
@@ -8,32 +8,43 @@ export const buildS3Storage = (
   accessKeyId: string,
   secretAccessKey: string
```

Comment:
should we also update this var name for consistency?

Reply:
Actually the name should be `secretAccessKey`, it's better to rename the `accessSecretKey` to `secretAccessKey`, but that is a breaking change, so no hurry to do it.

Reply:
cool. we may keep the backward compatibility and mark deprecated when changing and release all breaking changes when next major release is ready?

Reply:
would be great to have a comment or issue for this though

Reply:
LOG-6032